### DISCUSSION TITLE
Peak Calling

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1395] [Questions / Documentation] [answered]
Hi @rcorces ,

I was wondering if the score columns from the getPeakSet() output could be used to filter peaks? The reason is that when I look at the browser, some peaks have very very low accessibility, but they are still in the peak matrix. That influences my downstream work which is related to peak-to-gene analysis. I prefer not to include those peaks in the downstream analysis since I don't think they are that reliable. In your opinion, does it make sense to filter peaks based on the score?

Thank you!

**Comment by rcorces** [comment, marked as answer]
that makes sense to me. You could alternatively change the `cutOff` param to `addReproduciblePeakSet()`
